binder/environment.yml

```diff
@@ -7,5 +7,6 @@ dependencies:
 - isort
 - jupyterlab=0.35
 - yapf
+- rpy2
 - pip:
   - jupyterlab_code_formatter
```

Comment:
I see that you removed new line in another commit (b043c1c25f13725deb6b318c925066aa4997745e), you can probably drop that commit - it was a mistake to not have new line that I didn't spot before and I am okay with having it fixed in the first commit (c9339e8d0d966e2583f219b0350a67cb0cb57892)

Reply:
Good caught.